automationTest/src/test/java/com/owncloud/android/test/ui/models/LoginForm.java

```diff
@@ -0,0 +1,91 @@
+package com.owncloud.android.test.ui.models;
+
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.PageFactory;
+
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.AndroidElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.AppiumFieldDecorator;
+
+public class LoginForm {
+	final AndroidDriver driver;
+	
+	@CacheLookup
+	@AndroidFindBy(uiAutomator = "new UiSelector().description(\"Server address\")")
```

Comment:
If it is possible to use getResources I think is better to get the string "auth_host_address" instead of using the literal "Server address".  
For this case it will be: getResources().getString(R.string.auth_host_address)